cfgov/unprocessed/apps/analytics-gtm/js/util/analytics-util.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Check if an element exists on the page, and if it does, add listeners.
+ * @param {[type]}   elem     [description]
+ * @param {[type]}   event    [description]
+ * @param {Function} callback [description]
+ */
+function addEventListenerToElem( elem, event, callback ) {
+  if ( elem ) {
+    elem.addEventListener( event, callback );
+  } else {
+    analyticsLog( `${ elem } doesn't exist!` );
+  }
+}
+
+const delay = ( function() {
+  let timer = 0;
+  return function( callback, ms ) {
+    clearTimeout( timer );
+    timer = setTimeout( callback, ms );
+  };
+} )();
+
+/**
+ * TODO: Merge with Analytics.js.
+ * Track an analytics event and log the event.
+ * @param {string} event Type of event.
+ * @param {string} action Name of event.
+ * @param {string} label DOM element label.
+ */
+function track( event, action, label ) {
+  window.dataLayer.push( {
+    event: event,
+    action: action,
+    label: label
+  } );
+  analyticsLog( event, action, label );
+}
+
+/**
+ * Check if two hosts are the same.
+ * This only works with a www subdomain.
+ * @param {string} host1 - A URL.
+ * @param {string} host2 - Another URL.
+ * @returns {boolean} True if the hosts are equal, false otherwise.
+ */
+function hostsAreEqual( host1, host2 ) {
+  let h1 = document.createElement( 'a' );
```

Comment:
I would use an inner function so we don't have got repeat the code

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3957/commits/dafdc3f8a9c79c16892698ca535e01651dcb24ce